src/arr/compiler/pyret.arr

```diff
@@ -123,7 +126,10 @@ fun main(args :: List<String>) -> Number block:
         print-error("Passing command line arguments without compiling standalone no longer supported\n")
         failure-code
       else:
-        if r.has-key("build-runnable") block:
+        if r.has-key("auto-generate") block:
```

Comment:
I wish this were more parameterized, because others might have a use for this feature as well. For example, why not take in two paths with auto-generate, and make one be a source file and one be a destination file. Then if I wanted visitors for ast-anf, I might be able to generate those without writing any new Pyret code.